.travis.yml

```diff
@@ -73,10 +82,10 @@ matrix:
         - docker load -i $HOME/docker/images.tar || true
         - docker image list
         - docker build --cache-from ${GOOGLE_DOCKER_IMAGE} --build-arg COMMIT=$TRAVIS_COMMIT -t $IMAGE .
-        - make docker-push
-        - make pubsub-publish
         - docker tag $IMAGE ${GOOGLE_DOCKER_IMAGE}
         - IMAGE=${GOOGLE_DOCKER_IMAGE} make docker-push
+        - make pubsub-publish
```

Comment:
I guess here IMAGE var is required too https://github.com/athenianco/ci/blob/c30a1382435d024e188ecb3f6acd9e35339f9906/Makefile.main#L95